spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -859,7 +859,8 @@ def process_signatures(self, params):
                 line, _ = self.get_cursor_line_column()
                 active_parameter_idx = signature['activeParameter']
                 signature = signature['signatures']
-                func_doc = signature['documentation']
+                func_doc = signature['documentation'].replace(u'\xa0', ' ')

```

Comment:
This was generating an error on Python 2

Reply:
What kind of error?

Reply:
cannot decode `ascii` '\xa0'` (which are spaces)